bundles/ru.arsysop.svn.connector/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,15 @@
+Manifest-Version: 1.0
+Automatic-Module-Name: ru.arsysop.svn.connector
+Bundle-ManifestVersion: 2
+Bundle-SymbolicName: ru.arsysop.svn.connector
+Bundle-Version: 0.1.0.qualifier
+Export-Package: ru.arsysop.svn.connector
+Require-Bundle: org.eclipse.core.runtime;bundle-version="3.18.0",
+ org.eclipse.osgi.services;bundle-version="3.8.0";resolution:=optional,
+ org.eclipse.team.svn.core;bundle-version="5.0.0",
+ wrapped.org.tmatesoft.svnkit.svnkit;bundle-version="1.10.11",
```

Comment:
hmm
1. we can tune bundle symbolic name during wrapping to something nice
2. we should prefer `Import-Package` directive for 3rd party